runsc/cgroup/cgroup_v2.go

```diff
@@ -236,7 +236,16 @@ func (c *cgroupV2) CPUQuota() (float64, error) {
 	if err != nil {
 		return -1, err
 	}
-
+	cpuQuota, err := parseCPUQuota(cpuMax)
```

Comment:
Could you add a comment describing why also need to check the parent?

Reply:
Added description.